Sources/GoldenAccountsUseCase/GoldenSampleAppAccountsUseCase.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  GoldenSampleAppAccountsUseCase.swift
+//  GoldenSampleApp
+//
+//  Created by Backbase R&D B.V. on 12/10/2023.
+//
+
+import Foundation
+import AccountsJourney
+import ArrangementsClient2Gen2
+import ClientCommonGen2
+
+final class GoldenSampleAppAccountsUseCase: AccountsUseCase {
```

Comment:
@WingEraser & @DevAgani 

The very latest journeys don't deliver separate UseCases as separate module. This means we can have the UseCase contract and implementation in the same module

Reply:
@WingEraser wouldn't you agree that in a non Backbase setup, modular approach would be per feature or for the case of our customers, per journey? 

We technically already have that because each journey is built in R&D in its own project. For most customers it would likely make more sense to split module per feature. WDYT?

Reply:
@burgbart Direct me to one of the `latest journeys`?